embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/metadata/MetadataUtils.java

```diff
@@ -227,14 +226,16 @@ static boolean isJailbroken() {
      * @return true if the device is detected to be an emulator, false otherwise
      */
     static boolean isEmulator() {
-        boolean isEmulator = Build.FINGERPRINT.startsWith("generic")
-            || Build.FINGERPRINT.startsWith("unknown")
-            || Build.MODEL.contains("google_sdk")
-            || Build.MODEL.contains("Emulator")
-            || Build.MODEL.contains("Android SDK built for x86")
-            || Build.MANUFACTURER.contains("Genymotion")
-            || (Build.BRAND.startsWith("generic") && Build.DEVICE.startsWith("generic"))
-            || "google_sdk".equals(Build.PRODUCT);
+        boolean isEmulator = Build.FINGERPRINT.startsWith("generic") ||
```

Comment:
Using the latest [detection heuristic](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:benchmark/benchmark-common/src/main/java/androidx/benchmark/DeviceInfo.kt;l=29?q=emulator&ss=androidx%2Fplatform%2Fframeworks%2Fsupport:benchmark%2F) that AndroidX benchmark uses